src/aiida_pseudo/cli/install.py

```diff
@@ -148,118 +149,252 @@ def download_sssp(
+
+            with tarfile.open(tarball_path, 'w') as tarball_file:
+                for filepath in [filepath_configuration, filepath_metadata, filepath_archive]:
+                    tarball_file.add(filepath, filepath.name)
+            echo.echo_success(f'Pseudopotential archive written to: {tarball_path.name}')
             return
 
-        with open(filepath_metadata, 'rb') as handle:
-            handle.seek(0)
-            metadata = json.load(handle)
+        if QueryBuilder().append(SsspFamily, filters={'label': label}).first():
+            echo.echo_report(f'{SsspFamily.__name__}<{label}> is already installed')
+            sys.exit(1)
 
-        with attempt('unpacking archive and parsing pseudos... ', include_traceback=traceback):
-            family = create_family_from_archive(SsspFamily, label, filepath_archive)
+        description = (
+            f'SSSP v{configuration.version} {configuration.functional} {configuration.protocol} '
+            f'installed with aiida-pseudo v{__version__}'
+        )
+        description += f'\nArchive pseudos md5: {md5_file(filepath_archive)}'
+        description += f'\nPseudo metadata md5: {md5_file(filepath_metadata)}'
+
+        install_sssp(filepath_archive, filepath_metadata, label, description, traceback)
+
+
+def download_pseudo_dojo(
+    configuration: 'PseudoDojoConfiguration',
+    filepath_archive: pathlib.Path,
+    filepath_metadata: pathlib.Path,
+    traceback: bool = False
+) -> None:
+    """Download the pseudopotential archive and metadata for a PseudoDojo configuration to a path on disk.
+
+    :param configuration: the PseudoDojo configuration to download.
+    :param filepath_archive: absolute filepath to write the pseudopotential archive to.
+    :param filepath_metadata: absolute filepath to write the metadata archive to.
+    :param traceback: boolean, if true, print the traceback when an exception occurs.
+    """
+    from ..groups.family.pseudo_dojo import PseudoDojoFamily
+    from .utils import attempt
 
-        cutoffs = {}
+    label = PseudoDojoFamily.format_configuration_label(configuration)
+    url_archive = PseudoDojoFamily.get_url_archive(label)
+    url_metadata = PseudoDojoFamily.get_url_metadata(label)
+
+    with attempt('downloading selected pseudopotentials archive... ', include_traceback=traceback):
+        response = requests.get(url_archive, timeout=30)
+        response.raise_for_status()
+        with open(filepath_archive, 'wb') as handle:
+            handle.write(response.content)
+            handle.flush()
+
+    with attempt('downloading selected pseudopotentials metadata archive... ', include_traceback=traceback):
+        response = requests.get(url_metadata, timeout=30)
+        response.raise_for_status()
+        with open(filepath_metadata, 'wb') as handle:
+            handle.write(response.content)
+            handle.flush()
+
+
+def install_pseudo_dojo(
+    configuration: 'PseudoDojoConfiguration',
```

Comment:
Think this is not actual used
```suggestion
```

Reply:
It's used to see if the `configuration` is in the `paw_configurations`, see:

https://github.com/mbercx/aiida-pseudo/blob/3882e06d93884d34643959449b65b642c64f0eb4/src/aiida_pseudo/cli/install.py#L377